Final_project_2/frontend/css/style.css

```diff
@@ -485,26 +480,34 @@ address{
   margin-left: 80px;
   margin-bottom: 70px;
 }
+
 address .fa {
   display: block;
   font-size: 33px;
   margin-bottom: 20px;
 }
 
+.address-item {
```

Comment:
This is a new class I added to style the 3 items inside the address block (address, email, phone number). You can see that we use just a bunch of properties with the value `center` which means that they will stay in the center even if you reside the screen. If instead I'd have added like a `margin-left: 55px` then this would not be responsive in the same way.